pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -41,8 +43,13 @@ func NewStorage(s storage.Interface) *REST {
 
 		CreateStrategy: clusterpolicy.Strategy,
 		UpdateStrategy: clusterpolicy.Strategy,
+	}
 
-		Storage: s,
+	if opts, err := optsGetter.GetRESTOptions(store.QualifiedResource); err != nil {
+		glog.Fatal(err)
```

Comment:
An error would be a lot better.  There are very few resources I'd choose to kill the API server over.  I think it would be better to have the choice in a piece of higher level logic.